core/controllers/classifier_test.py

```diff
@@ -168,30 +168,31 @@ def setUp(self):
             secret, self.payload['message'])
 
     def test_next_job_handler(self):
-        json_response = self.post_json('/ml/nextjobhandler',
-                                       self.payload, expect_errors=False,
-                                       expected_status_int=200)
+        json_response = self.post_json(
+            r'/ml/nextjobhandler', self.payload, expect_errors=False)
```

Comment:
why raw string literals here and on line 176 but not at other places?

Reply:
My bad. It really does not matter whether we use raw strings or not. We can keep it or remove it, your call?

Reply:
@prasanna08 let's not block on this. You pick. Just be consistent.

Reply:
Done